crypto/helpers.go

```diff
@@ -152,7 +233,91 @@ func RSAEncrypt(plaintext []byte, publicKey *rsa.PublicKey) ([]byte, error) {
 	}
 }
 
-func RSADecrypt(ciphertext []byte, privateKey *rsa.PrivateKey) ([]byte, error) {
+func eciesEncrypt(plaintext []byte, publicKey *ecdsa.PublicKey) ([]byte, error) {
```

Comment:
Any particular reason for making these ec functions "private" (ie start with lowercase)?